src/machine/machinectl.c

```diff
@@ -1667,7 +1668,21 @@ static int enable_machine(int argc, char *argv[], void *userdata) {
                 goto finish;
         }
 
-        r = 0;
+        if (arg_now) {
+                _cleanup_strv_free_ char **new_args = NULL;
+
+                new_args = strv_new(streq(argv[0], "enable") ? "start" : "stop");
+                if (!new_args) {
+                        r = -ENOMEM;
```

Comment:
```c
r = log_oom()
```

i.e. we should log about all errors here, and given nothing logged here so far we should do so here